components/renku_data_services/authz/authz.py

```diff
@@ -117,6 +133,26 @@ async def get_project_users(
             else:
                 return MemberQualifier.SOME, users_list  # type: ignore[return-value]
 
+    async def get_project_users(self, requested_by: APIUser, project_id: str, scope: Scope) -> List[RoleUserId]:
```

Comment:
They are doing different things with the list of users. In one case, we are only interested in the list of users regardless of qualifiers (we remove every `id` that is `None`) while in the other case we are interested in the qualifiers. So, I had to add a flag to distinguish between the two. Having separate functions is cleaner compared to passing a flag.

I've put the common functionality in a function to avoid code duplication.